modules/core/src/utils/__tests__/string-utils.test.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { StringUtils } from "../string-utils";
+
+describe("Test isEqualCaseInsensitive function", () => {
```

Comment:
```suggestion
describe("Case insensitive string matching works as expected", () => {
```

Reply:
I have concerns about this change because, when unit test errors are displayed, it might confuse developers. The heading suggests that the testing method is working as expected, even though the tests are failing. Therefore, I believe it’s better to keep it as `Test isEqualCaseInsensitive function`, as it is both clear and self-explanatory.

Reply:
<img width="555" alt="image" src="https://github.com/user-attachments/assets/d7decbc3-1775-4d5a-84ac-02fccf4704b0" />